release_dashboard/lib/services/local_conductor.dart

```diff
@@ -32,8 +32,11 @@ class LocalConductorService extends ConductorService {
   Directory get rootDirectory => fs.directory(platform.environment['HOME']);
   File get stateFile => fs.file(defaultStateFilePath(platform));
 
-  static const String frameworkUpstream = 'https://github.com/flutter/flutter';
-  static const String engineUpstream = 'https://github.com/flutter/engine';
+  @override
+  late String engineCheckoutDirectory;
+
+  @override
+  late String frameworkCheckoutDirectory;
```

Comment:
Why are these returning `String` instead of `Directory`?

Reply:
+1, if you make it the directory, the caller can then call `service.frameworkCheckoutDirectory.path` themselves